src/javascript/_common/base/client_base.js

```diff
@@ -135,6 +135,9 @@ const ClientBase = (() => {
         return only_real_loginids.every(loginid => get('currency', loginid) && !isCryptocurrency(get('currency', loginid)));
     };
 
+    const isWalletsAccount = (loginid) => getAllAccountsObject()[loginid].account_category === 'wallet';
```

Comment:
Because it is not TS, it's hard to say if `loginid` is defined and if it's present among keys of returned object `getAllAccountsObject()`.

Reply:
Sorry. What do u mean to say?

[loginid] accepts anything

Reply:
I mean, is there opportunity to get `undefined` here `getAllAccountsObject()[loginid]`?

Reply:
fair point. i handled the edge cases. please check